### PR TITLE
Commit TF Upgrade changes. TF 0.12.16, AzureRM 2.20.0 - Master

### PR DESCRIPTION
**Pull request by vijayrajagopalan-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1910

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by shashank-rastogi — APPROVED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/AuthorisedServicesIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
```
Comment (shashank-rastogi) [inline src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/AuthorisedServicesIntegrationTest.java:5]: why are these java files not in master already? where r u merging this from?

**Review by shashank-rastogi — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.professionalapi.generator;
+
+public class ProfessionalApiGeneratorConstants {
+
```
Comment (shashank-rastogi) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java:4]: why are these java files not in master already? where r u merging this from?

File: src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/AuthorisedServicesIntegrationTest.java (hunk 1 above)
Comment (vijayrajagopalan-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/AuthorisedServicesIntegrationTest.java]: Synced master branch

File: src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java (hunk 2 above)
Comment (vijayrajagopalan-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java]: Synced master branch

**Review by shashank-rastogi — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java (hunk 2 above)
Comment (KOTLAPRA23) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/generator/ProfessionalApiGeneratorConstants.java]: removed the file

**Review by KOTLAPRA23 — APPROVED** [review]

**Pull request merged**